src/Appwrite/Utopia/Response/Model/HealthAntiVirus.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class HealthAntiVirus extends Model
```

Comment:
```suggestion
class HealthAntivirus extends Model
```
Change the filename as well to `HealthAntivirus.php`